composer.json

```diff
@@ -29,6 +28,13 @@
     }
   },
   "require-dev": {
+    "phpunit/phpunit": "^5.2",
```

Comment:
You're right. I edited that commit and kept the added test script in the composer json file